libraries/AP_IOMCU/iofirmware/iofirmware.cpp

```diff
@@ -942,7 +961,7 @@ bool AP_IOMCU_FW::handle_code_write()
             // no input when override is active
             break;
         }
-        if (rx_io_packet.count != sizeof(reg_direct_pwm.pwm)/2) {
+        if (rx_io_packet.count > sizeof(reg_direct_pwm.pwm)/2) {
```

Comment:
is AP_IOMCU ever sending less than a full set? if so, why?

Reply:
Yes - https://github.com/ArduPilot/ardupilot/blob/5c1ed968b18ea196b8376cdd6cc7aeae9b3085c4/libraries/AP_IOMCU/AP_IOMCU.cpp#L370 - don't know why as I didn't write this code.

Reply:
But I verified on the IOMCU side that this was happening